src/pages/ServiceList/ServiceListPage.tsx

```diff
@@ -14,24 +14,38 @@ type ServiceListProps = {
 class ServiceListPage extends React.Component<ServiceListProps, ServiceListState> {
   constructor(props: ServiceListProps) {
     super(props);
-
     console.log('Starting ServiceListPage');
+
+    this.dismissAlert = this.dismissAlert.bind(this);
```

Comment:
+1, but thinking out loud, I wonder if this part could be generalized at higher level (like in App.tsx) so that we don't have to repeat alert management everywhere.

Reply:
Probably, perhaps not for this iteration, but yes, a shared alert manager would be interesting.
This one is quite basic, just to not throw in the console the errors.